src/common/address.go

```diff
@@ -2,6 +2,7 @@ package common
 
 var (
 	// NodeNameDict is an optional dictionary to make node address human readable in log.
+	// TODO FIXIT: NodeNameDict is not populated.
```

Comment:
How do you mean to fix it?

Reply:
Either add code that populate it, either remove it.